tools/c7n_azure/c7n_azure/resources/sqlserver.py

```diff
@@ -136,6 +137,77 @@ def __call__(self, i):
         return super(AzureADAdministratorsFilter, self).__call__(i['properties']['administrators'])
 
 
+@SqlServer.filter_registry.register('vulnerability-assessment')
+class VulnerabilityAssessmentFilter(Filter):
+    """
+    Filter sql servers by whether they have recurring vulnerability scans
+    enabled.
+
+    :example:
+
+    Find SQL servers without vulnerability assessments enabled.
+
+    .. code-block:: yaml
+
+        policies:
+          - name: sql-server-no-va
+            resource: azure.sql-server
+            filters:
+              - type: vulnerability-assessment
+                enabled: false
+
+    """
+
+    schema = type_schema(
+        'vulnerability-assessment',
+        required=['type', 'enabled'],
+        **{
+            'enabled': {"type": "boolean"},
+        }
+    )
+
+    log = logging.getLogger('custodian.azure.sqldatabase.VulnerabilityAssessmentFilter')
```

Comment:
I know we have a mix of `camelCase` and `dash-case`.. Should we pick a standard one?

Reply:
its generally lower case for loggers re standard